modules/i18n/src/main/java/com/devonfw/module/i18n/common/util/I18nConstants.java

```diff
@@ -0,0 +1,86 @@
+package com.devonfw.module.i18n.common.util;
+
+/**
+ * Constants for i18n module
+ *
+ */
+public class I18nConstants {
+
+  /**
+   * Constant for the character DOT
+   */
+  public static final String DOT = ".";
+
+  /**
+   * Constant for the character 0
+   */
+  public static final int ZERO = 0;
+
+  /**
+   * Constant for the Error message in case of Invalid Locale
+   */
+  public static final String INVALID_LOCALE = "Invalid Locale . Please provide valid Locale";
+
+  /**
+   * Constants for Resource path for NLS bundle
+   *
+   * The files needed to be store in following package, and file name will be starting with NlsBundleI18n
+   *
+   * sample - com\devonfw\module\i18n\common\api\nls\NlsBundleI18n_en.properties
+   */
+  public static final String NLS_BUNDLE_INTF_NAME = "com/devonfw/module/i18n/common/api/nls/NlsBundleI18n";
+
+  /**
+   * Empty String
+   */
+  public static final String EMPTY_STRING = "";
+
+  /**
+   *
+   */
+  // public static final String LOCALE_FILES_LOCATION = "src/main/resources/locale/";
+
+  /**
+   * Constants for resource for other locale integration using gson option
+   *
+   * The files needed to be store in following package, and file name will be starting with messages
+   *
+   * Sample file - locale\messages_en_US.properties
+   */
+  public static final String RESOURCE_BASENAME = "locale/messages";
+
+  /**
+   * Closing bracket
+   */
+  public static final String CLOSING_BRACE = "}";
+
+  /**
+   * Opening bracket
+   */
+  public static final String OPENING_BRACE = "{";
```

Comment:
IMHO such stuff does not belong here:
1. it is an implementation pointlessly exposed to the public
2. it has nothing to do with I18n but is due to the fact that we decided to use JSON serialization here.